lua/entities/gmod_wire_expression2/core/custom/prop.lua

```diff
@@ -355,6 +355,41 @@ e2function string entity:propPhysicalMaterial()
 	return ""
 end
 
+hook.Add( "CanDrive", "checkPropStaticE2", function( ply, ent ) return ent.propStaticE2 == nil end )
```

Comment:
I believe this is implemented incorrectly. Currently, this hook will always return true/false, which means that any other CanDrive hooks added by other addons that happen to run after this hook will never run. Therefore, you might break other addons. You should only return something when necessary.